src/utils/CommunityModel.jl

```diff
@@ -0,0 +1,47 @@
+"""
+$(TYPEDSIGNATURES)
+
+Extract the semantic solution of the variables for a specific `community_member`
+from `res`. Removes the `community_member` prefix in the string ids of the
+returned dictionary.
+"""
+function values_community_member_dict(
+    semantics::Symbol,
+    res::ModelWithResult{<:Model},
+    community_member::CommunityMember,
+)
+
+    is_solved(res) || return nothing
+
+    val_d = values_community_member_dict(res, community_member)
+
+    (sem_ids, _, sem_vard, _) = Accessors.Internal.semantics(semantics)
+
+    ids = sem_ids(community_member.model)
+
+    Dict(
+        id => sum(v * val_d[k] for (k, v) in sem_vard(community_member.model)[id]) for
+        id in ids
+    )
+end
```

Comment:
- the sem_vard dict must be only constructed once (it's pricey)

also I think this is not a completely good way to approach this thing. Could we instead have a function like split_community_names and filter_member or so which allows the user to construct this themselves easily? As in, the function this way looks pretty good but it's very single-purpose-ish, and spends a lot of code on redoing stuff that could (should!) be done using the basic accessors.

Ideally, it could be
```
... |> values_dict(:reactions) |> split_community_members_dict
```
gets you
```
Dict(organism 1 => (reaction 1 => ..., reaction 2 => ...), organism 2 => ...)
```

or better
```
... |> filter_community_model_dict
```

Reply:
mostly done with this, but need `bind`